PSKoans/Koans/IntroTraining/AboutCmdletVerbs.koans.ps1

```diff
@@ -0,0 +1,198 @@
+
+            You'll see that there's only one line of text in the file. This is because the Set-Content command
+            will overwrite information that's already there.
+
+            Some Set-* cmdlets require the instance to already be present for you to change it;
+            you'll need to use a New-* cmdlet first to create an instance before you can 
+            overwrite information within it.
+
+            A common example one may see working with Office 365 is with calendar permissions. 
+            If a user already has some permissions configured, you can use you use "Set-MailboxFolderPermission"
+            to change the user's permissions. However, attempting to use an Add cmdlet will fail; 
+            Set-* commands overwrite data, but often can only do so if it already exists.
+
+        #>
+
+        It 'is for commands that overwrite data' {
+            <#
+                Using the Get-Command cmdlet, which retrieves all the cmdlets currently available,
+                find 5 new set commands you haven't previously used.
+            #>
+
+            # Replace each __ with the name of a Set cmdlet
+            $Answers = "____","____","____","____","____"
+            $Answers | Should -BeIn (Get-Command -Verb Set).Name
+
+            $UniqueCheck = $Answers | Get-Unique
+            $UniqueCheck.Count -eq 5 | Should -BeTrue -Because "five unique cmdlets should be supplied"
+
+        }
+
+    }
+
+    Context "Remove" {
+         <#
+            Cmdlets with the Remove verb will delete data from an object or data source.
+
+            If you followed the example in the 'New','Add' and 'Set' tests, you can use Remove-Item to 
+            delete your text file:
+            
+                $Path = "YOUR PATH.txt"
+                Remove-Item -Path $Path
+            
+            Before continuing, run this command a few times. What happens when you try to run it
+            once the instance has been deleted?
+
+            You'll see that it fails. Cmdlets with the verb Remove simply remove data;
+            if the instance you're referring to doesn't exist, then there's nothing available
+            to remove and it will emit an error message.
+
+            Returning to Calendar permissions in Office 365, you can use a Remove cmdlet
+            to completely remove a user's permissions to a calendar.
+        #>
+
+        It "is for commands that delete data" {
+            <#
+                Using the Get-Command cmdlet, which retrieves all available commands,
+                find 5 new remove commands you haven't previously used.
+            #>
+
+            # Replace each __ with the name of a Remove cmdlet
+            $Answers = "____","____","____","____","____"
+            $Answers | Should -BeIn (Get-Command -Verb Remove).Name
+
+            $UniqueCheck = $Answers | Get-Unique
+            $UniqueCheck.Count -eq 5 | Should -BeTrue -Because "five unique cmdlets should be supplied"
+
```

Comment:
There's a handful of these extra lines at the ends of sections; can you trim those out? Otherwise this file is good! Thank you! 😊

Reply:
I think I get what you mean, but I'm not too sure. I've committed a few changes 🦄